packages/typeahead/src/react/index.js

```diff
@@ -32,8 +32,10 @@ const Typeahead = React.forwardRef((props, forwardedRef) => {
   const portal = usePortal()
   const containerRef = React.useRef()
 
-  const fieldRef = React.useRef()
-  const inputRef = React.useRef()
+  const [target, setTarget] = useState()
+  const inputRef = React.useCallback(node => {
+    setTarget(node ? node.parentNode : undefined)
```

Comment:
I like the reason that the change was made: Staying with the React standard of a single value in a ref. 

The use of useCallback is pretty cool.  Didn't know you could do that.  

Seems like the solution is bound to the DOM hierarchy.  But that seems ok because it's internal hierarchy and this traversal could be changed if markup ever was.